config/deploy/staging.rb

```diff
@@ -0,0 +1,45 @@
+# Simple Role Syntax
+# ==================
+# Supports bulk-adding hosts to roles, the primary server in each group
+# is considered to be the first unless any hosts have the primary
+# property set.  Don't declare `role :all`, it's a meta role.
+
+role :app, %w{pramati@192.168.2.24}
+role :web, %w{192.168.2.24}
+#role :db,  %w{deploy@example.com}
+
+
+# Extended Server Syntax
+# ======================
+# This can be used to drop a more detailed server definition into the
+# server list. The second argument is a, or duck-types, Hash and is
+# used to set extended properties on the server.
+
+server '192.168.2.24', user: 'pramati', roles: %w{web,app}, my_property: :my_value
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.